README.md

```diff
@@ -26,7 +26,6 @@ If you are interested in releasing your app on the Contentful marketplace, pleas
 * [App SDK documentation](https://www.contentful.com/developers/docs/extensibility/ui-extensions/sdk-reference/)
 * [Management HTTP API reference documentation](https://www.contentful.com/developers/docs/references/content-management-api/)
 * [Marketplace](https://www.contentful.com/marketplace/)
-* [Marketplace early access program](https://docs.google.com/forms/d/e/1FAIpQLSeEuxxROsNHCq_zGRfmjwiAjKr06Mk92ubt0l0Dpi-0Z_0DmA/viewform)
```

Comment:
The link is also present under **Publishing your app**

Reply:
remove that section for now